.github/_typos.toml

```diff
@@ -18,6 +18,7 @@ extend-exclude = [
 
 [default.extend-words]
 ACI = "ACI" # Azure Container Instance
+exercize = "exercize" #test typos
```

Comment:
Can we ignore the misspelling just were needed so we don't end up with this misspelling elsewhere in the code

Reply:
Discussed in teams